IDEAS/Buildings/Components/OuterWall.mo

```diff
@@ -26,8 +28,16 @@ model OuterWall "Opaque building envelope construction"
   final parameter Real U_value=1/(1/8 + sum(constructionType.mats.R) + 1/25)
     "Wall U-value";
 
-  parameter Real coeffsCp[:,:]=[0,0.4; 45,0.1; 90,-0.3; 135,-0.35; 180,-0.2; 225,
-      -0.35; 270,-0.3; 315,0.1; 360,0.4]
+
+  replaceable parameter
+    IDEAS.Buildings.Data.WindPressureCoeff.Lowrise_Square_Exposed Cp_table
```

Comment:
Can you remove this from the result file? Using https://build.openmodelica.org/Documentation/ModelicaReference.Annotations.HideResult.html